packages/components/inputs/rich-text-utils/src/plugins/mark.tsx

```diff
@@ -35,20 +69,26 @@ const MarkPlugin = (options = {}) => {
   return [
     {
       // eslint-disable-next-line consistent-return
-      onKeyDown(event, editor, next) {
-        if (!isHotKey(event)) {
+      onKeyDown(
+        event: TEvent,
+        editor: { toggleMark: (arg0: unknown) => void },
+        next: () => void
+      ): void {
+        if (!isHotKey(event as KeyboardEvent)) {
           return next();
         }
 
         event.preventDefault();
         editor.toggleMark(options.typeName);
       },
-      renderMark(props, editor, next) {
+      //@ts-ignore
```

Comment:
why are not replace editor with an underscore and remove the `ts-ignore`

Reply:
If it is possible and if it won't break anything, It'll be nice to remove the editor parameter and arguments in the renderMark function and wherever it is used.